src/styles/index.ts

```diff
@@ -3207,6 +3207,20 @@ const styles = (theme: ThemeColors) =>
             marginLeft: 3,
         },
 
+        WorkspaceRightColumn: {
+            marginLeft: 100,
```

Comment:
Can you show me exactly what this looks like? I don't understand why we are hard coding so much margin. Does that mean we have 100px of space that can only be occupied by margin? How does that work when the viewport is bigger or smaller? It just feels like these hard coded margins are a hack and don't provide good responsiveness.